ui-components/date-time-pickers.md

```diff
@@ -0,0 +1,161 @@
+- Localization support.
+- Date range selection.
+- Customizable date and time formats.
+- Integration with MUI components.
+
+Documentation: https://mui.com/x/react-date-pickers/getting-started/
+
+---
+
+### React-DateTime-Picker
+
+**React-DateTime-Picker** is a flexible date and time picker library for React. It offers both date and time selection capabilities and is designed to be customizable to suit different project requirements.
+
+**Features:**
+
+- Support for selecting date, time, or both.
+- Customizable date and time formats.
+- Time interval configuration (e.g., 5-minute increments).
+- Ability to disable specific dates or time intervals.
+- Localization support.
+
+Documentation: https://projects.wojtekmaj.pl/react-datetime-picker
+
+---
+
+### React-Dates
+
+**React-Dates** is a powerful date picker library developed by Airbnb. It provides a collection of pre-built components to pick single dates, date ranges, and even allows selecting multiple dates.
+
+**Features:**
+
+- Date range selection with start and end dates.
+- Customizable calendar presentation and layout.
+- Support for choosing single dates and multiple dates.
+- Localization and internationalization support.
+- Ability to disable specific dates or date ranges.
+
+Documentation: https://github.com/airbnb/react-dates
+
+---
+
+### React-Day-Picker
+
+**React-Day-Picker** is a straightforward and lightweight date picker for React. It aims to be easy to use and easy to customize.
+
+**Features:**
+
+- Single date selection.
+- Customizable date format.
+- Ability to disable specific dates or date ranges.
+- Basic localization support.
+
+Documentation: https://github.com/gpbl/react-day-picker
+
+## Date utility libraries
+
+These libraries provide various functionalities to work with dates and time in React applications. Here are some popular date-related JS libraries:
+
+### Date-fns
+
+**date-fns** is a popular and comprehensive date utility library for JavaScript. It is designed to handle various common date and time-related operations, making it easier to work with dates in your JavaScript and React applications. Date-fns aims to be simple, functional, and focused on providing an intuitive API for working with dates.
+
+**Pros:**
+
+- Lightweight and tree-shakable, resulting in smaller bundle sizes.
+- Comprehensive functionality for date manipulation and formatting.
+- Follows functional programming principles, leading to predictable and immutable date operations.
+- Good TypeScript support with built-in type declarations.
+- Well-maintained and actively developed, with a growing community.
+
+**Cons:**
+
+- The API might be slightly different from Moment.js, which could require some adjustments if migrating from Moment.js.
+- Less popular compared to Moment.js (although popularity has been growing steadily).
+
+GitHub Repository: https://github.com/date-fns/date-fns
+Official Website: https://date-fns.org/
+
+---
+
+### Moment.js
```

Comment:
as I know the moment is deprecated, not sure we need use one in the new projects
WDYT?
I`d like to remove this section or mark it as **deprecated**

Reply:
Yeah, I didn't know about that). I'll change to [Luxon](https://moment.github.io/luxon/#/?id=luxon)